docs/sources/menu.yaml

```diff
@@ -83,119 +83,125 @@
+  - link: /features/panels/alertlist/
+    name: Alert list
+  - link: /features/panels/dashlist/
+    name: Dashboard list
+  - link: /features/panels/text/
+    name: Text
+- name: Dashboard features
+  link: /features/dashboard/
+  children:
+  - link: /features/dashboard/dashboards/
+    name: Overview
+  - link: /reference/annotations/
+    name: Annotations
+  - link: /reference/dashboard_folders/
+    name: Folders
+  - link: /reference/playlist/
+    name: Playlist
+  - link: /reference/search/
+    name: Search
+  - link: /reference/share_dashboard/
+    name: Sharing a dashboard
+  - link: /reference/share_panel/
+    name: Sharing a panel
+  - link: /reference/timerange/
+    name: Time range
+  - link: /reference/export_import/
+    name: Export and import
+  - link: /reference/datalinks/
+    name: Data links
+  - link: /reference/dashboard_history/
+    name: Dashboard version history
   - name: Keyboard Shortcuts
     link: /features/shortcuts/
   - name: Reporting
     link: /features/reporting/
-  - name: Navigation links
-    link: /features/navigation-links/
+  - link: /reference/dashboard/
+    name: JSON Model
+  - link: /reference/scripting/
+    name: Scripted dashboards
+- name: Data sources
+  link: /features/datasources/
+  children:
+  - link: /features/datasources/add-a-data-source/
+    name: Add data source
+  - link: /features/datasources/cloudwatch/
+    name: AWS Cloudwatch
+  - link: /features/datasources/azuremonitor/
+    name: Azure Monitor
+  - link: /features/datasources/elasticsearch/
+    name: Elasticsearch
+  - link: /features/datasources/stackdriver/
+    name: Google Stackdriver
+  - link: /features/datasources/graphite/
+    name: Graphite
+  - link: /features/datasources/influxdb/
+    name: InfluxDB
+  - link: /features/datasources/loki/
+    name: Loki
+  - link: /features/datasources/mssql/
+    name: Microsoft SQL Server
+  - link: /features/datasources/mixed/
+    name: MixedData
+  - link: /features/datasources/mysql/
+    name: MySQL
+  - link: /features/datasources/opentsdb/
+    name: OpenTSDB
+  - link: /features/datasources/postgres/
+    name: PostgreSQL
+  - link: /features/datasources/prometheus/
+    name: Prometheus
+  - link: /features/datasources/testdata/
+    name: TestData DB
+- name: Explore
+  link: /features/explore/
+- name: Alerting
+  link: /alerting/
+  children:
+  - link: /alerting/rules/
+    name: Engine and Rules
+  - link: /alerting/metrics/
+    name: Metrics
+  - link: /alerting/notifications/
+    name: Notifications
+- name: Image rendering
+  link: /administration/image_rendering/
+- name: Navigation links
+  link: /features/navigation-links/
+- name: Templates and variables
+  link: /variables/
+  children:
+    - link: /variables/templates-and-variables/
+      name: Overview
+    - link: /variables/global-variables/
+      name: Global variables
+    - link: /variables/advanced-variable-format-options/
+      name: Advanced variable format options
```

Comment:
This title might be a bit too long for a really deeply nested page. Advanced feels a bit unnecessary. Maybe just "Variable formating"

Reply:
I want to differentiate them from the general variable formatting, the syntax of which is covered in the main topic. Any recommendations?

Reply:
Also, changed it to "Advanced variable formats" in the menu. I do not love "advanced," but am at a loss for anything else.